wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -637,7 +637,7 @@ def user_display_name(user):
     user.get_full_name() if implemented and non-empty, or user.get_username() otherwise.
     """
     try:
-        full_name = user.get_full_name()
+        full_name = user.get_full_name().strip()
```

Comment:
Maybe worth adding this to `get_username` line as well for consistency? Admittedly you'd need to override the default validator to get spaces in the username, but it happens.

Reply:
The `strip()` is needed here because if `get_full_name` has been implemented naively as `return self.first_name + ' ' + self.last_name`, then a blank name will be returned as a single space, and it will fail to fall back on the username. There's no equivalent scenario where `get_username` would 'accidentally' return whitespace (and if it does, there's no further fallback we can provide other than returning that whitespace username) so I'd say `strip()` is unnecessary there.